terragrunt/README.md

```diff
@@ -1,20 +1,122 @@
 # Terragrunt
 
-This directory contains all of the infrastructure configuration controlled through terragrunt.
+This directory contains all the infrastructure configuration controlled through
+Terragrunt.
 
-## What and why terragrunt?
+## What and why Terragrunt?
 
-[Terragrunt](https://terragrunt.gruntwork.io/) is a wrapper around terraform that adds some additional functionality that we find useful. The features of terragrunt we use are:
-* Configuration of s3 and dynamodb to handle remote terraform state files. This is done automatically instead of needing to be configured manually as is the case when using plain terraform.
-* Module reuse. Terragrunt allows for different environments (e.g., staging and production) which share almost all of their configuration. When using plain terraform this would require copy/pasting configuration and updating configuration in multiple places.
+[Terragrunt] is a wrapper around [Terraform] that adds some additional
+functionality that we find useful. The features of Terragrunt we use are:
 
+  - [Managed state files](https://terragrunt.gruntwork.io/docs/getting-started/quick-start/#keep-your-backend-configuration-dry):
+    Terragrunt can create the S3 bucket and DynamoDB tables that Terraform needs
+    to store its state. Automating their creation ensures consistency across
+    environments.
+  - [Versioned environments](https://terragrunt.gruntwork.io/docs/getting-started/quick-start/#promote-immutable-versioned-terraform-modules-across-environments):
+    With Terragrunt, we can have multiple environments that use different
+    versions of a Terraform module. This enables us to test on staging, review
+    changes, and then update production to the specific version.
 
-## Directory structure
+## Directory Structure
 
-The `terragrunt` directory is split into two subdirectories:
-* `modules`: the reuseable module definitions for the logical services we manage.
-* `aws`: AWS account definitions which use the modules in the `modules` directory to configure the various AWS accounts we manage.
+The [Terragrunt] configuration is split into two parts: _modules_ and _state_:
 
-## Running
+- _Modules_ are normal [Terraform] modules that each provision a specific part
```

Comment:
I think we might not want to say that the "provision" but rather than the provide the configuration for provisioning. Then we should mention that _state_ use these modules as to do the provisioning.